clusterloader/cluster_loader.go

```diff
@@ -156,32 +158,16 @@ func createTemplate(baseName string, ns *v1.Namespace, configPath string, numObj
 		if tuning != nil {
 			if tuning.Templates.RateLimit.Delay != 0 {
 				framework.Logf("Sleeping %d ms between template creation.", tuning.Templates.RateLimit.Delay)
-				time.Sleep(time.Duration(tuning.Templates.RateLimit.Delay) * time.Millisecond)
+				time.Sleep(tuning.Templates.RateLimit.Delay * time.Millisecond)
```

Comment:
If you make the field in the config like "timeout_seconds" (or something explicitly mentioning units), then conversion is expected.
I just don't want to multiply duration by duration to get duration - this is really bad in my opinion.

But to be honest, I think that you should be able to set field with type duration to things like "1s", "10m" or sth like that (and this is the way we should proceed in my opinion).